gt/main.c

```diff
@@ -231,7 +421,15 @@ gt_proc(void *arg)
 			else
 				rte_pktmbuf_free(m);
 
-			/* TODO Reply the policy decision to GK-GT unit. */
+			/*
+			 * TODO Reply in a batch.
+			 * Reply the policy decision to GK-GT unit.
+			 */
+			ret = fill_notify_pkt(socket,
+				notify_pkt, &policy, &tunnel, gt_conf);
+			if (ret == 0 && rte_eth_tx_burst(
+					port, tx_queue, &notify_pkt, 1) != 1)
+				rte_pktmbuf_free(notify_pkt);
```

Comment:
`fill_notify_pkt()` allocates a packet, but it is never assigned to variable `notify_pkt`. So this call to `rte_pktmbuf_free()` is doing nothing at best.